certbot/certbot/_internal/renewal.py

```diff
@@ -347,52 +348,49 @@ def report(msgs, category):
 
 def _renew_describe_results(config, renew_successes, renew_failures,
                             renew_skipped, parse_failures):
+    # type: (interfaces.IConfig, List[str], List[str], List[str], List[str]) -> None
+    """
+    Print a report to the terminal about the results of the renewal process.
+    :param interfaces.IConfig config: Configuration
```

Comment:
```suggestion

    :param interfaces.IConfig config: Configuration
```

I know we need newlines in many weird places for things to format correctly; I honestly can't remember if this is one or not, but either way we do usually put a newline there.